example/package.json

```diff
@@ -1,16 +1,18 @@
 {
   "name": "example",
-  "version": "0.1.0",
+  "version": "0.2.0",
   "license": "MIT",
   "scripts": {
     "start": "styleguidist server"
   },
   "devDependencies": {
-    "prop-types": "15.5.10",
-    "react": "15.6.1",
-    "react-dom": "15.6.1",
-    "react-styleguidist": "5.4.4",
-    "snapguidist": "next",
-    "webpack": "1.14.0"
+    "babel-core": "6.26.0",
+    "babel-loader": "7.1.2",
+    "prop-types": "15.6.0",
+    "react": "16.2.0",
+    "react-dom": "16.2.0",
+    "react-styleguidist": "6.2.2",
+    "snapguidist": "./..",
```

Comment:
I had to change `snapguidist` version to "./.." to get the correct dependencies. It could be changed back to next after you publish if you want.